controllers/articles.controller.js

```diff
@@ -0,0 +1,23 @@
+const { readArticles, getArticleById }  = require('../models/articles.model');
+
+const getArticles = (request, response) => {
+    readArticles()
+    .then((articles) => {
+        response.status(200).send({articles})
+    })
+}
+
+const getArticleByIdController = (request, response) => {
+    const { article_id } = request.params;
+    getArticleById(article_id)
+    .then(({ rows, rowCount }) => {
+        if(rowCount === 0) {
+            response.status(404).send({ msg: 'Article not found'})
+        } else {
+            response.status(200).send({ articles : [rows[0]] })
+        }
```

Comment:
Here you are sending `[rows[0]]` which is wrapped in `[ ]` which is why the data you receive in the test is an array. If you send `rows[0]` it will just be the object which is preferable for an endpoint like this fetching a specific thing